temporal/t.rast.series/t.rast.series.py

```diff
@@ -62,6 +62,15 @@
 # % answer: start_time
 # %end
 
+# %option G_OPT_M_NPROCS
+# % description: Number of cores for multiprocessing in r.series
+# % answer: 1
```

Comment:
I suggest to remove these 2 lines. If you really want to keep the description one, it should be corrected, technically it's not multiprocessing but multi-threading, so "Number of cores used in r.series" would sound best to me.

Reply:
Agreed, thanks for the feedback!